deepchem/dock/tests/test_binding_pocket.py

```diff
@@ -149,35 +149,6 @@ def test_convex_find_pockets(self):
 
     assert len(pockets) < len(all_pockets)
 
-  def test_rf_convex_find_pockets(self):
```

Comment:
Ah, good find. Marking the tests as ignored seems like a good intermediate. There's been some good work on binding site prediction [recently](https://academic.oup.com/bioinformatics/article-abstract/doi/10.1093/bioinformatics/btx350/3859178/DeepSite-Protein-binding-site-predictor-using-3D?redirectedFrom=fulltext) which I'd like to get into DeepChem. Keeping these older tests around will be useful, even if we need to fix them up later.